### PR TITLE
change port for local development to 5080

### PR DESCRIPTION
**Pull request by xuwhite**
because of apple!!!

### CODE REVIEW

**Comment by MyIgel** [comment]
Any opinion on that @marudor / @weeman1337 ?
Eventually having something with 80 (which is not 8080) would be more explicit.

File: docker/dev/docker-compose.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ services:
       APP_NAME: Engelsystem DEV
     env_file: deployment.env
     ports:
-      - "5000:80"
+      - "5001:80"
```
Comment (MyIgel) [inline docker/dev/docker-compose.yml:23]: To be consistent that should be changed in `docker/docker-compose.yml` too (and added to the dev documentation as it is missing there ^^).

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: docker/dev/docker-compose.yml (hunk 1 above)
Comment (xuwhite) [reply docker/dev/docker-compose.yml:23]: done

**Comment by weeman1337** [comment]
`5080`?

**Pull request merged**